app/javascript/pages/TaxCenter/Index.tsx

```diff
@@ -1,12 +1,3 @@
-import { usePage } from "@inertiajs/react";
-import React from "react";
+import TaxCenterPage from "$app/components/TaxCenter";
 
-import TaxCenterPage, { type TaxCenterPageProps } from "$app/components/TaxCenter";
-
-function index() {
-  const props = usePage<TaxCenterPageProps>().props;
-
-  return <TaxCenterPage {...props} />;
-}
-
-export default index;
+export default TaxCenterPage;
```

Comment:
Shouldn't you need to retrieve the props with `usePage` directly in `TaxCenterPage`? :thinking: 
My suggestion was to inline the TaxCenterPage's code directly in the index function instead of calling for a component elsewhere, but perhaps you have something different in mind?

Reply:
Oh, I followed `app/javascript/pages/Collaborators/Index.tsx` structure assuming that's how you meant it to be.

It works as-is, BTW. But if you prefer, can inline the `components/TaxCenter.tsx` code into this `pages/TaxCenter/Index.tsx` as well. LMK.